boards/cats_rev1Pro/Core/Src/control/kalman_filter.c

```diff
@@ -508,7 +528,7 @@ cats_error_e kalman_step(kalman_filter_t *filter, state_estimation_data_t *data,
   if (elimination->num_faulty_baros > 1) {
     log_error("Kalman step faulty baros: %d", elimination->num_faulty_baros);
   }
-  if (elimination->num_faulty_imus > 1) {
+  if (elimination->num_faulty_imus >= 1) {
```

Comment:
Should we log faulty imus every iteration?

Reply:
I think, this is just for debugging, the errors are not logged here on the flash. But yes, the actual error logging should only happen once in the sensor_elimination file. Will open an issue